examples/resources/alkira_connector_gcp_interconnect/resource1.tf

```diff
@@ -0,0 +1,36 @@
+resource "alkira_connector_gcp_interconnect" "example_gcp_interconnect_1" {
+  name              = "example_gcp_interconnect_1"
+  size              = "LARGE"
+  description       = "example connector with multiple instances"
+  cxp               = "US-WEST"
+  group             = alkira_group.group1.name
+  tunnel_protocol   = "IPSEC"
+  loopback_prefixes = ["10.40.0.0/24"]
+  instances {
+    name                     = "instance1"
+    edge_availability_domain = "AVAILABILITY_DOMAIN_1"
+    customer_asn             = 56009
+    bgp_auth_key             = "key"
+  }
+
+  instances {
+    name                     = "instance2"
+    edge_availability_domain = "AVAILABILITY_DOMAIN_1"
+    customer_asn             = 56010
+    bgp_auth_key             = "key_2"
+  }
+  segment_options {
+    segment_id               = alkira_segment.segment1.id
+    instance_name            = "instance1"
+    advertise_on_prem_routes = true
+    disable_internet_exit    = false
+    customer_gateways {
+      loopback_ip  = "192.192.192.192"
```

Comment:
@asim-ak remove loopback_ip from examples. we put this behind a flag and don't want the customers to use this for now. We will enable to only on customer ask and if absolutely required.